### PR TITLE
remove the duplicated and combine them

### PR DESCRIPTION
**Pull request by LillyWu**

### CODE REVIEW

File: content/docs/setup/kubernetes/platform-setup/index.md [hunk 1]
```diff
@@ -7,21 +7,6 @@ keywords: [kubernetes]
 
 Follow these instructions to setup the Kubernetes cluster for Istio.
 
-## Prerequisites
```
Comment (sdake) [inline content/docs/setup/kubernetes/platform-setup/index.md]: can you revert this part of the PR?  I prefer the prerequisites for the kubernetes versions in platform-setup as it reads more smoothly.

File: content/docs/setup/kubernetes/quick-start/index.md [hunk 2]
```diff
@@ -25,327 +25,6 @@ cluster.
   > it completely before installing the newer version. Remember to uninstall
   > the Istio sidecar for all Istio enabled application pods too.
 
-### Setup Minikube
```
Comment (sdake) [inline content/docs/setup/kubernetes/quick-start/index.md]: removing these sections makes sense, assuming they are converged into platform-setup.

**Comment by sdake** [comment]
@LillyWu Thank you for the contribution!  This recently popped up as a problem in a prior review.  Thanks for taking the time to repair the problem.

@andraxylia you recently mentioned someone had merged some work here that duplicated information.  I've made some guesses about what you were after, but could you provide a review please so the intent is clear?

Cheers
-steve

**Comment by sdake** [comment]
@lillywu just noticed https://github.com/istio/istio.github.io/pull/1948 - Seems like @andra may be further along in achieving a desirable outcome.  Would you be kind enough to consolidate your work on top of that PR once it merges?

Cheers
-steve

File: content/docs/setup/kubernetes/platform-setup/index.md (hunk 1 above)
Comment (sdake) [reply content/docs/setup/kubernetes/platform-setup/index.md]: makes sense.  I'm not sure what state this PR is in given all the construction around platform setup.  I added a needs-rebase, so any remaining work items you wanted to tackle can be consolidated on top of master.  Please assign to me and I will process quickly.

Comment (LillyWu) [reply content/docs/setup/kubernetes/platform-setup/index.md]: The prerequisites have been removed from the master branch. And I think it looks fine. I plan to close my PR

**Comment by LillyWu** [comment]
Close this PR as it's duplicated

**Pull request closed without merging**